src/icons/clarity-icons.scss

```diff
@@ -50,6 +51,9 @@ clr-icon {
     &.icon-color-success {
         @include setIconColor($clr-icon-color-success);
     }
+    &.icon-color-error {
+        @include setIconColor($clr-icon-color-error);
```

Comment:
It seems like there is an opportunity to DRY this code out here. I will approve for now. But we should try to consolidate, instead of replicate.